tests/test_ellipsoidal_density_projection.py

```diff
@@ -19,16 +19,17 @@ def add_default_params(frame):
 
 
 TEST_SINGLE_FRAME = add_default_params(
-    ase.Atoms(symbols=["X"], positions=np.zeros((1, 3)), cell=(10, 10, 10), pbc=False)
+    ase.Atoms(symbols=["X"], positions=np.zeros((1, 3)), cell=(0, 0, 0), pbc=False)
 )
 TEST_QUAT_FRAME = add_default_params(
-    ase.Atoms(
-        symbols=["X", "O"], positions=[np.zeros(3), np.ones(3)], cell=(10, 10, 10)
-    )
+    ase.Atoms(symbols=["X", "O"], positions=[np.zeros(3), np.ones(3)], cell=(0, 0, 0))
 )
 TEST_MATRIX_FRAME = TEST_SINGLE_FRAME.copy()
 TEST_MATRIX_FRAME.arrays["matrix"] = [np.eye(3)]
 
+print(TEST_SINGLE_FRAME.arrays)
+print(TEST_QUAT_FRAME.arrays)
```

Comment:
are these print statements needed?

Reply:
Just remove these print statements and then I will approve.

Reply:
Done!